pkg/apiserver/clusterinfo/fetch.go

```diff
@@ -35,6 +35,7 @@ type ClusterInfo struct {
 	} `json:"pd"`
 	Grafana      *clusterinfo.GrafanaInfo      `json:"grafana"`
 	AlertManager *clusterinfo.AlertManagerInfo `json:"alert_manager"`
+	AlertCount   *int                          `json:"alert_count"`
```

Comment:
I prefer to use another structure and interface for alert count, because it is not used elsewhere.